src/containers/auth/login/index.js

```diff
@@ -79,7 +82,7 @@ class Login extends Component {
               onChange={this.onChange}
             />
             {errors.email && (
-              <span className={styles.invalidFeedback}>{errors.email}</span>
+              <span className={styles.invalid_feedback}>{errors.email}</span>
```

Comment:
Maybe it useful to push this functionality to a stateless component?

Reply:
You suggest to create stateless component for one `<span>`?

Reply:
Maybe it better to create HOC to make such notifications. This idea can have the rights to live. Maybe it can make the code shorter. It's just my thinking about it